README.md

```diff
@@ -121,243 +97,3 @@ While we are primarily reachable on our <a href="https://discourse.pi-hole.net/"
- ```
-1483964299 IPv4 www.google.com 2.3.4.5 2
-```
- Variant: `>getallqueries-domain www.google.com (15)` (see notes above)
-
-- `>getallqueries-client 2.3.4.5` : get all queries that FTL has in its database for a specific client name *or* IP
- ```
-1483964299 IPv4 www.google.com 2.3.4.5 2
-1483964302 IPv4 www.googleadservices.com 2.3.4.5 1
-1483964312 IPv4 www.gstatic.com 2.3.4.5 2
-1483964333 IPv4 www.linguee.de 2.3.4.5 2
-```
- Variant: `>getallqueries-client 2.3.4.5 (12)` (see notes above)
-
-- `>recentBlocked` : get most recently pi-holed domain name
- ```
-www.googleadservices.com
-```
- Variant: `>recentBlocked (4)` show the four most recent vlocked domains
-
-- `>memory` : get information about `FTL`'s memory usage due to its internal data structure
- ```
-memory allocated for internal data structure: 178688 bytes (178.69 KB)
-dynamically allocated allocated memory used for strings: 15084 bytes (15.08 KB)
-Sum: 193772 bytes (193.77 KB)
-```
-
-- `>clientID` : Get ID of currently connected client
- ```
-6
-```
-
-- `>version` : Get version information of the currently running FTL instance
- ```
- version v1.6-3-g106498d-dirty
-tag v1.6
-branch master
-date 2017-03-26 13:10:43 +0200
-```
-
-- `>dbstats` : Get some statistics about the FTL long-term storage database
- ```
- queries in database: 88387
-SQLite version: 3.19.3
-```
-
-### Debugging `pihole-FTL`
-
-#### Debugging instructions (when `FTL` is not starting reliably)
-
-Once you are used to it, you can skip most of the steps and debugging is actually quite easy and gives you insights into how software (not limited to `pihole-FTL`) works.
-
-1. Install `screen` and `gdb` (probably `sudo apt-get install screen gdb`)
-2. Start a screen session (it will allow you to come back even if the SSH connection died)
-  * If you don't know about `screen`, then read about it (you *will* love it!)
-3. Start a screen session using `screen`
-4. Ensure that `pihole-FTL` is terminated (e.g. `sudo killall pihole-FTL`)
-5. Arrange file permissions to be able to start `FTL` as your current user (the following assumes you are logged in as user `pi`).
-  * `sudo touch /var/log/pihole-FTL.log /run/pihole-FTL.pid /run/pihole-FTL.port`
-  * `sudo chown pi:pi /var/log/pihole-FTL.log /run/pihole-FTL.pid /run/pihole-FTL.port`
-  * `sudo chmod 0644 /var/log/pihole-FTL.log /run/pihole-FTL.pid /run/pihole-FTL.port`
-6. Start `pihole-FTL` in the debugger: `gdb pihole-FTL`
-7. Type `handle SIGHUP nostop SIGPIPE nostop` to instruct the debugger to don't stop on expected signals
-8. Type `run debug` to start `FTL` (you should see some lines of text and `FTL` should start successfully)
-9. You can now close the terminal (Ctrl+A and then D to detach) and come back later using (`screen -r`) when it has crashed
-10. If it has crashed, copy&paste the terminal output, and
-11. type also `backtrace` and post the output in a (new) issue
-12. We might ask for additional information in order to isolate your particular issue
```

Comment:
These debugging instructions when FTL is not starting reliably is not in the documentation atm.